README.md

```diff
@@ -126,6 +127,16 @@ If you don't have permission for global system installation, try the following _
                             positions to consider a gene to be present (default
                             15, or 1/3 of average sample coverage assessed by true
                             coverage analysis) (default: None)
+      --bowtieAlgo="--very-sensitive-local"
+                            Bowtie2 alignment mode. It can be an end-to-end alignment
+                            (unclipped alignment) or local alignment (soft clipped
+                            alignment). Also, can choose between fast or sensitive
+                            alignments. Please check Bowtie2 manual for extra information:
+                            http://bowtie-bio.sourceforge.net/bowtie2/index.shtml .
+                            This option should be provided between quotes and starting
+                            with an empty space (like --bowtieAlgo " --very-fast") or
+                            using equal sign (like --bowtieAlgo="--very-fast")
+                            (default: "--very-sensitive-local")
```

Comment:
You state that the option should be provided between quotes and starting with an empty space, but the default value isn't like this. Is this correct?

Reply:
The default only displays the default value that will be used